src/filesystem/implementation.js

```diff
@@ -278,6 +278,14 @@ function find_node(context, path, callback) {
   }
 }
 
+/**
+ * find_lnode
+ */
+// in: file or directory path
+// out: node structure, or error
+function find_symbolic_node(context, path, callback) {
+  lstat_file(context, path, callback);
```

Comment:
I think you want to move the implementation from `lstat_file` into this function and re-implement `lstat_file` using this function instead.